sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -203,7 +203,7 @@ def test_pinjoint_arbitrary_axis():
     assert A.ang_vel_in(N).magnitude() == sqrt(omega**2)
     angle = A.ang_vel_in(N).angle_between(A.x)
     assert angle.xreplace({omega: 1}) == 0
-    assert C.masscenter.vel(N).simplify() == - omega*N.z
+    assert C.masscenter.vel(N) == 0
```

Comment:
N is the reference frame of body P. The mass center of body C is at the joint because it wasn't defined. So the velocity should be zero. Makes sense.